cds_dojson/marc21/models/books/multipart.py

```diff
@@ -20,21 +20,22 @@
 from __future__ import unicode_literals
 
 from ..base import model as cds_base
+from .base import COMMON_IGNORE_FIELDS, CDSOverdoBookBase
 from .base import model as books_base
-from .base import CDSOverdoBookBase, COMMON_IGNORE_FIELDS
 
 
 class CDSMultipart(CDSOverdoBookBase):
     """Translation Index for CDS Books."""
 
     __query__ = '(690C_:BOOK OR 690C_:"YELLOW REPORT" OR ' \
-                '690C_:BOOKSUGGESTION OR 980__:PROCEEDINGS OR 980__:PERI OR ' \
+                '980__:PROCEEDINGS OR ' \
```

Comment:
for the `BOOKSUGGESTION`, we will delete them from CDS???

Reply:
in one of the discussion I as told by Anne that we don't need to migrate this collection